### DISCUSSION TITLE
Question about tile gaps / intermittent culling

### DISCUSSION THREAD

**Discussion by Revmatek** [crest-ocean/crest#1135] [General]
@daleeidd 

I'm pretty sure this is a bug, but I have no insight other than I see it occasionally and I am unable to capture it. It started occurring at some point after upgrading to 4.17 (I think). Very rarely, I see an entire slice of the ocean (sliver even, doesn't even seem like a tile) disappear and then reappear basically in the next frame (can't really tell). This happens over the course of hours in the editor and I have seen it at runtime as well. I have also seen an entire tile disappear and then reappear while in the editor. It would be difficult to capture on video but I thought maybe you would have some better insight presented with this information.

Seems similar to this (though I think unrelated to floating origins):
https://github.com/wave-harmonic/crest/issues/665
<img width="809" alt="image" src="https://github.com/wave-harmonic/crest/assets/114962753/e15f28be-3be9-4355-90f5-81e9198bdbf1">

Also I believe when I see a gap / slice disappear it is vertical relative to the perspective, not horizontal but I cannot be totally sure.

**Comment by Revmatek** [comment]
I am attempting to capture this on video, I just saw it in the boat wakes scene on latest. 

No luck... seems random.

**Comment by magenta404** [comment]
That's unfortunate… This is something that can happen due to precision issues. There are mitigations in place but I guess it hasn't covered everything.

I doubt it was introduced in 4.17 as those mitigations haven't been touched well before then. It could be related to a change in your Ocean Renderer quality settings. Or whether you are now further from world zero (may need Floating Origin). Generally, incorrect settings will produce persistent problems so this seems more like a bug.

Previous mitigation attempt is #595 and the improved attempt is #720. The latter has a clue to potentially solving it.

I will need more information about your Ocean Renderer and your camera's position at the time it happened. If it is possible to record gameplay (think recording positions of everything etc) then that could be a good way to capture the problem. Otherwise may need to guess which can do too.

Feel free to use "Create an issue from this discussion".

**Reply by Revmatek** [reply]
This happens in the BoatWakesScene in latest. I don't think any of my settings are in play here. I never started seeing this until after I upgraded fixing various other issues and floating origin is handled. I can try do record the problem in the BoatWakeScene again, though I sat there for about an hour trying to get it to happen again.

**Reply by Revmatek** [reply]
I have opened this issue as suggested:
https://github.com/wave-harmonic/crest/issues/1136

**Comment by Revmatek** [comment]
I am fairly certain the only thing that changed with my OceanRenderer after upgrading was changing LOD 7->8 due to LOD / clipping issues
